### DISCUSSION TITLE
Ventoy Update Hangs At 12% Every Time

### DISCUSSION THREAD

**Discussion by zepfan1965** [ventoy/Ventoy#1118] [General]
This issue started happening at some point within the last couple months or so, not sure exactly when. Every time I attempt to update Ventoy to a new release, ventoy2disk hangs at 12%. This happens 100% of the time and the only thing I can do is to close the program through Task Manager. I've tried several different USB drives, all with the same result. Rebooting the machine makes no difference. This is on Windows 10, 64 bit. The only way I've been able to update to a new release is to start over, formatting the drive, fresh install of Ventoy and then recopying all ISO files back to the drive. Any ideas?

**Comment by steve6375** [comment]
Try a different usb drive.

**Reply by zepfan1965** [reply]
I've already tried 3 or 4 different ones. I don't have another one at the moment to try, but since it did the same thing on the ones I've tried it seems logical to me that the drives are not the problem.

**Reply by steve6375** [reply]
Ok, maybe antivirus software is causing the problem?

**Comment by ventoy** [comment]
@zepfan1965  What about the latest release?

**Comment by mightymo77** [comment]
I tried updating from version 1.0.54 on two different flash drives, and two different ports. It gets to 12% for a little while, will jump to 22%, and then hang. Have to kill the process. Tried rebooting. Flash drives and Windows 10 installation are fine.

**Reply by ventoy** [reply]
Try latest 1.0.55

**Reply by mightymo77** [reply]
Yes this occurred on version 1.0.55, trying to upgrade from both 1.0.54 and 1.0.53 on two different flash drives.

**Reply by ventoy** [reply]
Please upload the log.txt

**Reply by mightymo77** [reply]
Log file attached. Currently testing flash drive (128GB Lexar) with FakeFlashTest v1.1.2.

**Reply by mightymo77** [reply]
Finished with FakeFlashTest:

Test Empty Space (slow):
"The unused file space on this drive volume seems good, however to ensure the memory is not faulty, I recommend you run the Quick Size Test."

I am sure my drives are fine.

**Reply by ventoy** [reply]
Please use this Ventoy2Disk.exe to replace the one in Ventoy-1.0.55 and retry and then upload the log.txt again.
https://github.com/ventoy/Ventoy/blob/master/INSTALL/Ventoy2Disk.exe

**Reply by mightymo77** [reply]
Same thing with the new exe, stuck at 22%.

**Reply by ventoy** [reply]
There are many process running in your computer? In the log it says find 112853 processes.

**Reply by mightymo77** [reply]
What? The log is wrong. I have Task Manager opened right now, I have 93 processes running.

![Screenshot 2021-10-20 111759](https://user-images.githubusercontent.com/57172042/138121750-292bbcb7-a0b5-49db-b640-83260a0b677a.png)

**Reply by ventoy** [reply]
My mistake.
Please download and try again (I update it):
https://github.com/ventoy/Ventoy/blob/master/INSTALL/Ventoy2Disk.exe

**Reply by mightymo77** [reply]
Used that second new exe. This time I received a failed error message from Ventoy.

**Reply by ventoy** [reply]
It seems the USB is locked by some process, but can't find which one.
Maybe some antrVirus.
Try disable all "Windows Security" options (Realtime, Cloud-Delivered, Automatic Sample & Tamper protection.

Or use VentoyLiveCD to update: 
https://www.ventoy.net/en/doc_fail_update.html

**Reply by mightymo77** [reply]
OK the VentoyLiveCD processed worked just fine. You should try to make an English version of that.

This is an excellent tool. Thank you very much for developing it.

**Reply by steve6375** [reply]
The write errors in the log look like access is blocked by Antivirus/antimalware software. Check task manager Services for Acronis protection, Kaspersky, etc.

**Reply by mightymo77** [reply]
I only have Windows Security (Windows 10) installed, I have no third party anti-virus software.

**Reply by ventoy** [reply]
Please try the newest
https://github.com/ventoy/Ventoy/blob/master/INSTALL/Ventoy2Disk.exe

**Reply by mightymo77** [reply]
Yes this newest exe worked. What did you change?

**Reply by ventoy** [reply]
I need the log.txt for analyse.

**Reply by mightymo77** [reply]
Here is the log.
<deleted>

**Comment by steve6375** [comment]
AntiVirus may be blocking access. Disable AV.
Log.txt file is produced - what does it say and attach it.
Maybe all your USB Flash drives are fake? Are they all the same make/source?
e.g. a 1TB USB flash drive for $10 is going to be a fake.
You can test them using FakeFlashTest.exe or H2TESTW.

**Comment by ventoy** [comment]
Can latest 1.0.56 work?

**Reply by mightymo77** [reply]
No, same thing. It goes from 18%, to 22%, then produces an error message. Here is the log.

**Reply by ventoy** [reply]
What about this CI release:
https://github.com/ventoy/Ventoy/actions/runs/1375023619

**Reply by mightymo77** [reply]
Yes that version worked. I successfully upgraded my second flash drive from v1.0.55 to v10230810.

**Reply by ventoy** [reply]
OK, also plesae attach log.txt for analyse.

**Reply by mightymo77** [reply]
Log uploaded. Please let me know your findings.

**Reply by ventoy** [reply]
OK. There maybe some process that hold the disk and prevent Ventoy upgrade.
In the CI release I remove the EFI partition and recreate it. seems works.

**Reply by mightymo77** [reply]
OK yes, after this update the EFI drive opened in File Explorer. Are you going to use this method in final builds from now on?

**Reply by ventoy** [reply]
Do you mean after update the 32MB VTOYFEI partition can open in File Explorer?

**Reply by mightymo77** [reply]
Oh I just meant that when Ventoy recreated the EFI drive, it automatically popped open in File Explorer (due to my autostart settings). You can ignore that comment. But I am asking, that change that you made in the CI release, to where you remove the EFI partition and then recreate it, now that you know it works, will you now use that moving forward?

**Comment by GIBbeer** [comment]
Hi, same problem here - on my primary Win11 laptop I can't create Ventoy disk - hangs on 12%, but on my old Win7 headless terminal it works good. Log file attached.
Flash drive is 100% legit, no file hooks to the flash drive exists in File Locksmith PowerToy. Windows Defender is disabled. Tried few older Ventoy versions and nothing works.
[log.txt](https://github.com/ventoy/Ventoy/files/11068602/log.txt)
